### DISCUSSION TITLE
Discussing Final Project

### DISCUSSION THREAD

**Discussion by nggih** [OpenMined/courses#124] [Project: Privacy Product Specification] [answered]
Hi, thank you for making such a wonderful course. I can feel the passion and the enthusiasm on solving the privacy problem from Andrew, Emma and all the speakers. I would like to ask about the final project, 

- is it okay to discuss about the details of the final project/product here in the Discussion Forum? because I would like to make sure that my description of the product makes sense
- Regarding the attempts, are we going to get the feedback of each attempts? 
- should we change the product every attempt or just improving the previous attempt product?

thank you

**Comment by yemikifouly** [comment, marked as answer]
Hello Linggih! 👋 

I am really glad you are enjoying the course and making progress! 🙂 About your questions...

1. Absolutely! Feel free to discuss the details of the projects in the discussion board. Ideally, if you have questions about it you should post under this category: [Project: Privacy Product Specification](https://github.com/OpenMined/courses/discussions/categories/project-privacy-product-specification) and if you wish to have a more open-ended discussion about it with the other students you could post under [General](https://github.com/OpenMined/courses/discussions/categories/general).
2. Yes. You will get feedback after each attempt. So here is how it would work. The project will be pass or fail. Every time you submit a mentor will review it and provide you with feedback. If you did everything correctly, then you will pass the project and won't be able to resubmit. If you didn't, the mentor will tell you exactly what you are missing or doing incorrectly in your submission. You will receive this feedback and will be able to know what to change in order to pass when you resubmit.
3. No. You won't need to change the product for each attempt. Most of the time, it would be that you missed something or didn't do something correctly and you will just have to fix that. You won't have to change the product or even the project as a whole at all. 

I hope these help. 🙂
